bundles/org.openhab.binding.avmfritz/src/main/java/org/openhab/binding/avmfritz/internal/handler/BoxHandler.java

```diff
@@ -35,4 +58,45 @@ public BoxHandler(Bridge bridge, HttpClient httpClient,
             AVMFritzDynamicCommandDescriptionProvider commandDescriptionProvider) {
         super(bridge, httpClient, commandDescriptionProvider);
     }
+
+    @Override
+    protected void initConnections() {
+        AVMFritzBoxConfiguration config = getConfigAs(AVMFritzBoxConfiguration.class);
+        if (this.callMonitor == null && (isLinked(AVMFritzBindingConstants.CHANNEL_CALL_ACTIVE)
```

Comment:
You can reuse the `callChannelsLinked()` method here. Maybe store its result in local variable and access it later.

Reply:
Indeed, I had moved that method from another class and forgot to make use of it here. As there are only 5 channels on this bridge, I don't cache the result as I don't think it has a big impact.